.github/actions/upload-datadog/action.yaml

```diff
@@ -21,8 +23,32 @@ runs:
           echo "No API key provided, skipping..."
           exit 0
         fi
-        npm install -g @datadog/datadog-ci@2.21.0
-        datadog-ci junit upload --service coder ./gotests.xml \
+
+        TMP_DIR=$(mktemp -d)
+
+        if [[ "${{ runner.os }}" == "Windows" ]]; then
+          BINARY_PATH="${TMP_DIR}/datadog-ci.exe"
+          BINARY_URL="https://github.com/DataDog/datadog-ci/releases/download/v2.48.0/datadog-ci_win-x64"
```

Comment:
Thoughts on making the version an input to the action?
It would make it easy to bump the version in the future.